src/main/java/duke/Command/AddCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.Command;
+
+import duke.*;
```

Comment:
I think perhaps can consider importing individual classes instead of using wildcard `import`!